.github/workflows/github_actions.yml

```
@@ -11,7 +11,7 @@ jobs:
     - run: |
         python3 -m pip install black
     - run: |
-        python3 -m black --check ntrans.py ntrans_dataprep.py
+        python3 -m black --check ntrans.py ntrans_dataprep.py ntrans_combine.py
```

It seems like we always want to black all python files, so we might as well specify that instead of listing file names:

```suggestion
        python3 -m black --check *.py
```